src/lib/utils.ts

```diff
@@ -111,3 +111,32 @@ export function formatDateToDDMMYYYY(date?: Date | null) {
         new Date(date)
     );
 }
+
+/**
+ * Parses the HTML string and returns an array of sections.
+ * @param html - The HTML string to parse.
+ * @returns An array of sections.
+ */
+export function parseMarkdownToFaqs(html: string): FaqSection[] {
```

Comment:
we could try having this function a bit more generic so h2 & h3 are not the only headings allowed. You have an example in the assembly repo 🙏🏼 https://github.com/boxfish-studio/assembly-website/blob/main/src/components/indexed-section.svelte#L36